src/mapper.ts

```diff
@@ -64,19 +65,35 @@ export class Mapper {
           };
           title = this._templateSrv.replace(alias, scopedVars);
         }
+        const values = row.filter((value, idx) => !_.includes(skipIndexes, idx));
+        const mappedValues = this._mapNullValues(values, nullMapping);
```

Comment:
I guess, we should also use the mapped values earlier, when counting the max value. Can't add a comment there, it's line 52 in this file